WBooks/Extensions/Fonts.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  Fonts.swift
+//  WBooks
+//
+//  Created by daniel.f.munoz on 13/08/21.
+//
+
+import Foundation
+import UIKit
+
+extension UIFont {
+    static var booksTitle: UIFont {
```

Comment:
make the name more generic. the idea is to have the most used fonts accesible, if you name it booksTitle it seems like this font is specifically created for book title labels.

I suggest something like this:

```
extension UIFont {
    static var gillSansTitle: UIFont {
        return UIFont.gillSans(size: 16)
    }

    static var gillSansSubtitle: UIFont {
        return UIFont.gillSans(size: 14)
    }
    
    static func gillSans(size: CGFloat) -> UIFont {
        return UIFont(name: "Gill Sans", size: size)!
    }
}
```

Reply:
Done